apps-rendering/src/components/layout/standard.stories.tsx

```diff
@@ -1,24 +1,47 @@
 // ----- Imports ----- //
-import { ArticleDesign, ArticleDisplay, ArticlePillar } from '@guardian/libs';
+import type { ArticleFormat } from '@guardian/libs';
+import { ArticleDisplay } from '@guardian/libs';
 import { breakpoints } from '@guardian/source-foundations';
-import { partition } from '@guardian/types';
+import type { Option } from '@guardian/types';
+import { OptionKind, partition, some } from '@guardian/types';
+import Comment from 'components/layout/comment';
 import Standard from 'components/layout/standard';
-import { article, matchReport, review } from 'fixtures/item';
+import {
+	analysis,
+	article,
+	comment,
+	editorial,
+	letter,
+	matchReport,
+	review,
+} from 'fixtures/item';
+import { deadBlog, live } from 'fixtures/live';
+import type { Item } from 'item';
+import type { ReactElement } from 'react';
 import { renderAll } from 'renderer';
+import Live from './live';
 
-// ----- Stories ----- //
+// ----- Functions ----- //
+
+const formatFromItem = (
```

Comment:
I made this so we can optionally override the `item`'s display type by passing it explicitly. This way we can reuse the same fixture but force different display types (`Showcase`, `Immersive`, etc). 
I think as a long-term solution we should probably have automatically generated fixtures like DCR to showcase more variety but maybe for the time being this is ok?